load_data.py

```diff
@@ -1,9 +1,16 @@
 from tensorflow.contrib.learn.python.learn.datasets.mnist import extract_images, extract_labels
 import cPickle
 import numpy as np
+from os import environ, path, listdir
+import tensorflow as tf
 
 
-def load_mnist(file_name, dtype, file_path="./data/mnist/"):
+DATA_DIR = './data'
+if 'DATA_DIRECTORY' in environ:
+    DATA_DIR = environ['DATA_DIRECTORY']
```

Comment:
Why is this necessary?

Reply:
I think it's better not to have to type './data' all the time since a refactor would be more annoying.

Reply:
I agree :+1: . But we do not use the environment variable, do we? We always use the string defined at the top. Or am I blind?

Reply:
If you source the script it should export the variable. It's the last line in the script.

Reply:
Ah yea, found it. Very sweet I like it.